geode-core/src/main/java/org/apache/geode/cache/query/internal/DefaultQueryService.java

```diff
@@ -90,16 +91,17 @@
    */
   public static final boolean QUERY_HETEROGENEOUS_OBJECTS = Boolean
       .valueOf(System.getProperty(
-          DistributionConfig.GEMFIRE_PREFIX + "QueryService.QueryHeterogeneousObjects", "true"))
+          GEMFIRE_PREFIX + "QueryService.QueryHeterogeneousObjects", "true"))
       .booleanValue();
 
   public static final boolean COPY_ON_READ_AT_ENTRY_LEVEL = Boolean
       .valueOf(System.getProperty(
-          DistributionConfig.GEMFIRE_PREFIX + "QueryService.CopyOnReadAtEntryLevel", "false"))
+          GEMFIRE_PREFIX + "QueryService.CopyOnReadAtEntryLevel", "false"))
       .booleanValue();
 
+  @Deprecated
```

Comment:
just a thought: Maybe add a comment about what to use instead here as well?

Reply:
Done